frontend/src/components/urlInput/urlInput.tsx

```diff
@@ -50,7 +50,12 @@ const URLInput: FC<URLInputProps> = ({ control, type, errors }) => {
     const regex = new RegExp(regexStr);
     const match = regex.exec(url);
 
-    return match?.[1];
+    if (match.length > 1) {
+      match.shift();
```

Comment:
The first match is just the full matched string, everything after it is the capture groups. So we drop the first item since it is not relevant to our join.

Reply:
there are possible improvements to this code,
but in terms of possible bug - [`match` could be `null`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec#return_value), you need to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to prevent an exception:
```ts
    if (match?.length > 1) {
```